metacat-common-server/src/main/java/com/netflix/metacat/common/server/util/DataSourceManager.java

```diff
@@ -108,6 +123,47 @@ private synchronized void createDataSource(final String catalogName, final Map p
     }
 
     /**
+     * init properties.
+     *
+     * @throws Exception exception
+     */
+    @PostConstruct
+    public void init() throws Exception {
```

Comment:
We can talk about this but I don't quite understand why you don't just use this class and Inject it into the Mysql configuration classes and then just modify this within them. This seems backwards.